purchases/consumer-rules.pro

```diff
@@ -1,4 +1,5 @@
 -keep class com.revenuecat.** { *; }
 -keep class androidx.lifecycle.DefaultLifecycleObserver
 -dontwarn com.google.errorprone.annotations.CanIgnoreReturnValue
--dontwarn com.google.errorprone.annotations.Immutable
\ No newline at end of file
+-dontwarn com.google.errorprone.annotations.Immutable
+-dontwarn com.amazon.**
```

Comment:
As mentioned, we don't need to `keep` the amazon classes since they aren't actually available in this module. Those are added in the `feature/amazon` module, which does keep the amazon classes.